test/run_envoy_bazel_coverage.sh

```diff
@@ -80,7 +80,7 @@ rm "${SRCDIR}"/test/coverage/BUILD
 if [ "$VALIDATE_COVERAGE" == "true" ]
 then
   COVERAGE_VALUE=$(grep -Po 'lines: \K(\d|\.)*' "${COVERAGE_SUMMARY}")
-  COVERAGE_THRESHOLD=98.0
+  COVERAGE_THRESHOLD=97.5
```

Comment:
This is unfortunate. I guess there is no way around this in terms of some other gcovr exclusion type?

Reply:
Yeah with new GCC it includes more line, which led this. I think we may bring back this to 98.0 without much hack, but it is too much for this PR. I didn't try much with hacking gcovr, @htuch any idea?

Reply:
With bazel native coverage, I think it will drop anyway. We're still trying to debug this thought CC @iirina.